package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@f5devcentral/f5-fast-core",
-  "version": "0.19.1",
+  "version": "0.20.0",
```

Comment:
We can just do a patch version for updating dependencies as long as we did not update any major versions. In other words, a simple `npm update && npm audit --fix` can just be a patch release, but using `npm install` to move a dependency to a new major version should at least be a minor version change.